arch/arm/boot/dts/overlays/revpi-core-overlay.dts

```diff
@@ -103,11 +103,14 @@
 				brcm,function = <BCM2835_FSEL_ALT2>;
 				brcm,pull     = <BCM2835_PUD_OFF>;
 			};
-			sniff_a_pins: pb_sniff_a_pins {
-				/* A2 */
-				brcm,pins     = <28>;
+			sniff_pins: pb_sniff_pins {
```

Comment:
Maybe use sniff_pins instead of pb_sniff_pins. This would be more consistent to what we have at other places.

Reply:
ACK